File: 백준/Bronze/10808. 알파벳 개수/알파벳 개수.swift
var alphabetDictionary = [Character : Int]()
Array("abcdefghijklmnopqrstuvwxyz").forEach {
    alphabetDictionary[$0] = 0
}

Array(readLine()!).forEach {
    alphabetDictionary[$0, default:0] += 1
}

var result = alphabetDictionary
.sorted { $0.key < $1.key }
.reduce("") { $0 + "\($1.value) " }
result.removeLast()

print(result)

File: 백준/Bronze/10808. 알파벳 개수/알파벳 개수.py
alphabet = "abcdefghijklmnopqrstuvwxyz"
alphabet_dictionary = {}
for character in alphabet:
    alphabet_dictionary[character] = 0

word_inputed = input()

for character in word_inputed:
    alphabet_dictionary[character] += 1
    
print(" ".join([str(x[1]) for x in sorted(alphabet_dictionary.items())]))